boards/arm/serpente/serpente.yaml

```diff
@@ -0,0 +1,10 @@
+identifier: serpente
+name: SERPENTE
+type: mcu
+arch: arm
+ram: 32
+flash: 256
+toolchain:
```

Comment:
Consider adding a 'supported' section to get more build coverage.  See the `seeeduino_xiao` for an example.